src/components/Misson.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/prop-types */
+import PropTypes from 'prop-types';
```

Comment:
- [ ] Let's try to not disable Linter checks in our program as they are there to help us write better and more professional code. Disabling Linter checks defies its purpose and doesn't help you become a professional developer. Learning how to fix them now will set you up ready for the industry as that's the standard. Please fix it 👍🏽